app/views/courses/StudentRankingView.coffee

```diff
@@ -0,0 +1,198 @@
+helper = require 'lib/coursesHelper'
+require('app/styles/courses/student-ranking-view.sass')
+RootView = require 'views/core/RootView'
+template = require 'templates/courses/student-ranking-view'
+LevelSession = require 'models/LevelSession'
+CocoCollection = require 'collections/CocoCollection'
+Campaign = require 'models/Campaign'
+Level = require 'models/Level'
+utils = require 'core/utils'
+require 'three'
```

Comment:
Is this required?

Reply:
`#Level = require 'models/Level'` and `#require 'three'` are expendable. I'll remove them at the next commit.